src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let mens;
+
+  if (!century) {
+    mens = people.filter(person => person.sex === 'm');
+  } else {
+    mens = people.filter(person => person.sex === 'm' && Math.ceil(person.died / 100) === century);
```

Comment:
Посмотри, у тебя 2 раза полностью повторяется строка
```suggestion
    mens = people.filter(person => person.sex === 'm' && Math.ceil(person.died / 100) === century);
```
Давай сделаем так чтобы это было написано 1 раз, а уже внутри будем проверять, задано ли century или нет. В зависимости от этого возвращать то или иное значение